en/editor/publish/build-guide.md

```diff
@@ -0,0 +1,186 @@
+# Build Process and FAQ
+
+The build process is mainly divided into two parts, **General Build Process** and **Platform Adaptation Process**. Adaptive processing logic for each platform will be embedded in the **Build** panel as a separate plugin. The build plugin system is then open and developers can dynamically embed some build parameters into the panel for use.
+
+## General build Process
+
+The general build process for Cocos Creator 3D consists of the following:
+
+1. Initialization of build parameters
+2. Collation of build data
+3. Write the built resource to the file system
+4. Organizing the data of `settings`
+5. Compression and writing of `settings uuid`
+
+### Initialization of build parameters
+
+This step is mainly to initialize the initial `options` passed to the build to the internal `options` of the build, do some parameter formatting and initialize the resource data of the build resource database, load the latest resource information and classify it.
+
+### Collation of build data
+
+The editor will first summarize the scene currently involved in the build and all resources in the `resources` directory. Each resource is packaged through the engine's deserialization to find the dependent resource and recursion to pack the resources. The entire project's scripting environment is configured before being deserialized, that is, all non-plugin project scripts are loaded. Because whether the script loads correctly or not directly affects the deserialization, failure to load because the script is not written legally will directly result in build failure. If the dependent resource is lost in the deserialization process, a warning is issued, but the build continues nonetheless. The warning here does not mean that the problem does not need to be resolved, and if the resource loss is not resolved, it is difficult to guarantee that the problem will not occur after the build.
+
+This step will also sort out the resource types based on the build's internal division, such as scenes, scripts, texture compression tasks, JSON grouping information, etc., and weed out resource information that is not used. **Note**: All user scripts are loaded before this step is performed.
+
+### Write the built resource to the file system
+
+After performing the first two steps, then we need to generate the used resources into the file system. After build, all serialized files are placed in the `res/import` directory generated after the build, and the source files of all resources are placed in the `res/raw-assets` directory. This can be divided into the following steps:
```

Comment:
@yanOO1497 感觉还是原来的翻译比较合适一些？这段的中文意思是：

执行完前两个步骤后，将使用到的资源生成到文件系统内，打包之后，所有的序列化文件都会放置在构建后目录的 `res/import` 目录下，所有的资源的源文件将会放置在 `res/raw-assets` 目录下。具体可以分为以下几个步骤：

Reply:
After performing the first two steps, then we need to generate the used resources into the file system. After build, all serialized files are placed in the res/import directory generated after the build, and the source files of all resources are placed in the res/raw-assets directory. This can be divided into the following steps:
->
After performing the previous steps, then we need to generate the used resources into the file system. After building, the serialized JSON files of all resources are placed in the `res/import` directory. The original files of all resources are placed in the `res/raw-assets` directory. The build process can be broken down into the following phases:

这段话更能表达我原来中文想的意思，如果有语法问题可以再修改，句意大致就是这样了

Reply:
@yanOO1497 - agreed.

Reply:
已修改